run

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-exec gunicorn --log-level debug -w 4 -b 127.0.0.1:8000 teabot_endpoints.endpoints:app
+exec gunicorn --log-level debug -w 4 -b 0.0.0.0:${PORT:-8000} teabot_endpoints.endpoints:app
```

Comment:
I would assume that if gunicorn is using all of its workers then it will
just not call accept () on any other incoming sockets until the next worker
becomes available. This would mean that it will block for a fraction of a
second, but not drop any requests. I thought nginx queueing was mostly
about avoiding slowloris attacks tying up all of your workers by
drip-feeding request headers (I would assume that heroku's load balancer
also protects us from this, but I've not checked).